py/janitor/site/templates/generic/done.html

```diff
@@ -7,7 +7,7 @@
     {% endif %}
 {% endblock sidebar %}
 {% block page_title %}
-    Changes that have been merged or pushed
+    Done
```

Comment:
Please revert - this is less understandable than the previous description

Reply:
Done